front/admin/cli.ts

```diff
@@ -508,9 +544,13 @@ const main = async () => {
       return transcripts(command, argv);
     case "registry":
       return registry(command);
+    case "production-check":
+      return productionCheck(command, argv);
     default:
       console.log(
-        "Unknown object type, possible values: `workspace`, `user`, `data-source`, `event-schema`, `conversation`, `transcripts`"
+        "Unknown object type, possible values: " +
+          "`workspace`, `user`, `data-source`, `event-schema`, `conversation`, " +
+          "`transcripts`, `registry`, `production-check`"
```

Comment:
nit: maybe it's an opportunity to type objectType (so the error message could be dynamically generated and we could assertNever in the switch case).

Reply:
Will give it 1mn and otherwise move forward :)

Reply:
(done)